src/workloads/query/multiplanner/Subplanning.yml

```diff
@@ -0,0 +1,357 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  The goal of this test is to exercise subplanning. We create as many indexes as possible, and run a
+  query that makes all of them eligible, so we get as many competing plans as possible.
+
+  The original goal of this test was to demonstrate weaknesses of the SBE multiplanner when compared to
```

Comment:
Could you update this with a more specific description for this workload?

Reply:
I don't think that this is necessary.
The query in the workload is pretty straight-forward and I don't want to have to change description every time we tweak the query.

Reply:
I have added the query description